config/custom_navajo.json

```diff
@@ -26,13 +26,13 @@
     "expiry": "2023-09-30"
   },
   {
-    "url": "https://github.com/ocaml/ocaml/archive/5.0.0.tar.gz",
+    "url": "https://github.com/ocaml/ocaml/archive/refs/heads/5.0.tar.gz",
     "name": "5.0.0+stable+release",
-    "expiry": "2023-09-17"
+    "expiry": "2023-09-22"
```

Comment:
Can we extend this to next week so that we don't get into this one-off error as we did previously?

Reply:
Done!